qa/pull-tester/rpc-tests.py

```diff
@@ -172,6 +172,7 @@
     'bip9-softforks.py',
     'p2p-feefilter.py',
     'rpcbind_test.py',
+    'p2p-timeouts.py',
```

Comment:
micronit: Technically speaking, this test takes at least 62 seconds.\, so it should be moved up a bit.